pdaltools/standardize_format.py

```diff
@@ -61,20 +71,46 @@ def get_writer_parameters(new_parameters: Dict) -> Dict:
     override the standard ones
     """
     params = STANDARD_PARAMETERS | new_parameters
-
     return params
 
 
-def rewrite_with_pdal(input_file: str, output_file: str, params_from_parser: Dict) -> None:
+def remove_points_from_class(points, class_points_removed: []) :
+    input_dimensions = list(points.dtype.fields.keys())
+    dim_class = input_dimensions.index("Classification")
+
+    indice_pts_delete = [id for id in range(0, len(points)) if points[id][dim_class] in class_points_removed]
+    points_preserved = np.delete(points, indice_pts_delete)
+
+    if len(points_preserved) == 0:
+        raise Exception("All points removed !")
+
+    return points_preserved
+
+
+def rewrite_with_pdal(input_file: str, output_file: str, params_from_parser: Dict, class_points_removed: []) -> None:
     # Update parameters with command line values
-    params = get_writer_parameters(params_from_parser)
-    pipeline = pdal.Reader.las(input_file)
-    pipeline |= pdal.Writer(filename=output_file, forward="all", **params)
+    pipeline = pdal.Pipeline()
+    pipeline |= pdal.Reader.las(input_file)
     pipeline.execute()
+    points = pipeline.arrays[0]
+
+    if class_points_removed:
+        points = remove_points_from_class(points, class_points_removed)
+
+    #ToDo : it seems that the forward="all" doesn't work because we use a new pipeline
```

Comment:
Super le commentaire, peux tu préciser les metadatas concernés, comme ça :

"since we create a new pipeline, the 2 metadatas creation_doy and creation_year are update to current date instead of forwarded from input LAS"